src/ui/components/NetworkMonitor/ResponseBody.tsx

```diff
@@ -0,0 +1,54 @@
+import { ResponseBodyData } from "@recordreplay/protocol";
+import React, { useState } from "react";
+import ReactJson from "react-json-view";
+import { TriangleToggle } from "./RequestDetails";
+import { RequestSummary } from "./utils";
+
+const FIVE_MEGABYTES = 5e6;
+
+const safeJSONParseBase64 = (maybeJSON: string) => {
+  try {
+    return [true, JSON.parse(atob(maybeJSON))];
+  } catch (e) {
+    console.log(maybeJSON);
+    console.log(e);
+    return [false, maybeJSON];
+  }
+};
+
+const ResponseBody = ({
+  request,
+  responseBodyParts,
+}: {
+  request?: RequestSummary;
+  responseBodyParts: ResponseBodyData[];
+}) => {
+  if (!request || !responseBodyParts?.length) {
+    return null;
+  }
+
+  const contentLength = request.requestHeaders.find(h => h.name.toLowerCase() === "content-length");
```

Comment:
This header is the length of the request, not the response.

Reply:
Doh!